specification/chaos/resource-manager/Microsoft.Chaos/preview/2023-04-15-preview/types/experiments.json

```diff
@@ -327,15 +327,14 @@
     "selector": {
       "type": "object",
       "description": "Model that represents a selector in the Experiment resource.",
+      "discriminator": "type",
       "properties": {
         "type": {
           "type": "string",
           "description": "Enum of the selector type.",
           "enum": [
-            "Percent",
-            "Random",
-            "Tag",
-            "List"
+            "List",
```

Comment:
Previously, `List` was the only value supported by Chaos Studio. These previous existing enum values were never actually supported, and were added a long time ago.